README.md

```diff
@@ -30,281 +30,19 @@
-| Byte String, Binary Encoded Data | `[]byte`                                                                                                    | `[]byte{0x01, 0x02}`                                                                   |
-| Text String                      | `string`                                                                                                    | `"Hello, World!"`                                                                      |
-| Map                              | `map[any]any`                                                                                               | `map[string]float64{"one": 1.0}`                                                       |
-| Table name                       | `surrealdb.Table(name)`                                                                                     | `surrealdb.Table("users")`                                                             |
-| Record ID                        | `surrealdb.RecordID{Table: string, ID: any}`                                                                | `surrealdb.RecordID{Table: "customers", ID: 1}, surrealdb.NewRecordID("customers", 1)` |
-| Geometry Point                   | `surrealdb.GeometryPoint{Latitude: float64, Longitude: float64}`                                            | `surrealdb.GeometryPoint{Latitude: 11.11, Longitude: 22.22`                            |
-| Geometry Line                    | `surrealdb.GeometryLine{GeometricPoint1, GeometricPoint2,... }`                                             |                                                                                        |
-| Geometry Polygon                 | `surrealdb.GeometryPolygon{GeometryLine1, GeometryLine2,... }`                                              |                                                                                        |
-| Geometry Multipoint              | `surrealdb.GeometryMultiPoint{GeometryPoint1, GeometryPoint2,... }`                                         |                                                                                        |
-| Geometry MultiLine               | `surrealdb.GeometryMultiLine{GeometryLine1, GeometryLine2,... }`                                            |                                                                                        |
-| Geometry MultiPolygon            | `surrealdb.GeometryMultiPolygon{GeometryPolygon1, GeometryPolygon2,... }`                                   |                                                                                        |
-| Geometry Collection              | `surrealdb.GeometryMultiPolygon{GeometryPolygon1, GeometryLine2, GeometryPoint3, GeometryMultiPoint4,... }` |                                                                                        |
-
-## Helper Types
-
-### surrealdb.O
```

Comment:
Removed because it does not exist. Apparently we have `surrealdb.Obj` but I deprecated it in this pull request because it does't make much sense. You could use `map[string]any` instead of `surrealdb.Obj`- the former is 14 chars while the latter is 13 chars, but we won't need a type alias just to save one char.